homeassistant/components/denonavr/media_player.py

```diff
@@ -74,92 +89,115 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             unique_id = f"{config_entry.unique_id}-{receiver_zone.zone}"
         else:
             unique_id = None
```

Comment:
```suggestion
            unique_id = f"{config_entry.entry_id}-{receiver_zone.zone}"
```

Reply:
https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-of-last-resort

Reply:
that's easy 😄